test/end-to-end/specs/test_viewBlueprint.js

```diff
@@ -16,7 +17,8 @@ describe('View Blueprint Page', () => {
     beforeEach(() => {
       CreateBlueprintPage.newBlueprint(testData.blueprint.simple);
 
-      helper.goto(viewBlueprintPage)
+      browser
+        .click(BlueprintPage.blueprintNameSelector(testData.blueprint.simple.name))
```

Comment:
This is also more in line in how a user would use the page, so good improvement.